frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-inventory-card.tsx

```diff
@@ -54,24 +73,38 @@ export const VMInventoryCard: React.FC<VMInventoryCardProps> = () => {
         <DashboardCardTitle>Inventory</DashboardCardTitle>
       </DashboardCardHeader>
       <DashboardCardBody isLoading={isLoading}>
-        <InventoryItem
-          isLoading={isLoading}
-          title="NIC"
-          count={nicCount}
-          TitleComponent={NicsTitle}
-        />
-        <InventoryItem
-          isLoading={isLoading}
-          title="Disk"
-          count={diskCount}
-          TitleComponent={DisksTitle}
-        />
-        <InventoryItem
-          isLoading={isLoading}
-          title="CD-ROM"
-          count={cdromCount}
-          TitleComponent={CDROMTitle}
-        />
+        {nicCount > 0 && (
+          <InventoryItem
+            isLoading={isLoading}
+            title="NIC"
+            count={nicCount}
+            TitleComponent={NicsTitle}
+          />
+        )}
+        {diskCount > 0 && (
+          <InventoryItem
+            isLoading={isLoading}
+            title="Disk"
```

Comment:
Could we use toString methods of DiskType for the titles as well?

Reply:
nice :-) done